warp10/src/main/java/io/warp10/standalone/Warp.java

```diff
@@ -138,16 +139,51 @@ public static void main(String[] args) throws Exception {
 
     Properties properties = getProperties();
 
+    backend = properties.getProperty(Configuration.BACKEND);
+
+    boolean useLevelDB = false;
+    boolean useFDB = false;
+    boolean nullbackend = false;
+    boolean plasmabackend = false;
+    boolean inmemory = false;
+
     boolean analyticsEngineOnly = "true".equals(properties.getProperty(Configuration.ANALYTICS_ENGINE_ONLY));
-    boolean nullbackend = "true".equals(properties.getProperty(NULL)) || analyticsEngineOnly;
 
-    boolean plasmabackend = "true".equals(properties.getProperty(Configuration.PURE_PLASMA));
+    if (analyticsEngineOnly) {
+      nullbackend = true;
+      if (null != backend && !Constants.BACKEND_NULL.equals(backend)) {
+        LOG.warn("Backend specification '" + backend + "' ignored since '" + Configuration.ANALYTICS_ENGINE_ONLY + " is true.");
+      }
+    } else {
+      if (null == backend) {
+        throw new RuntimeException("Backend specification '" + backend + "' MUST be set since Warp 10 3.0");
+      }
```

Comment:
"Backend specification '" + **Configuration.BACKEND** + "' MUST be set since Warp 10 3.0"

Message may also include the thee main possible choices Constants.BACKEND_LEVELDB , Constants.BACKEND_FDB , Constants.BACKEND_MEMORY

Reply:
> "Backend specification '" + **Configuration.BACKEND** + "' MUST be set since Warp 10 3.0"

 done

> Message may also include the thee main possible choices Constants.BACKEND_LEVELDB , Constants.BACKEND_FDB , Constants.BACKEND_MEMORY

Not fan of that type of message, the user needs to know what s/he's doing.